cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/DocumentManager.cs

```diff
@@ -273,9 +273,14 @@ public IEnumerable<Document> GetDriverAndCaseDocuments(string caseId, string log
         {
             var documents = new List<bcgov_documenturl>();
 
-            documents.AddRange(dynamicsContext.bcgov_documenturls.Expand(doc => doc.bcgov_CaseId).Expand(doc => doc.dfp_DocumentTypeID)
+            /*documents.AddRange(dynamicsContext.bcgov_documenturls.Expand(doc => doc.bcgov_CaseId).Expand(doc => doc.dfp_DocumentTypeID)
```

Comment:
better to delete code like this to to have commented out code everywhere. If you need to see previous code you can always use git history

Reply:
Deleted the code which is commented out